kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -299,18 +300,34 @@ function showLearnChannel(store, channelId) {
   cookiejs.set('currentChannel', channelId);
   ContentNodeResource.setChannel(channelId);
 
-  const recommendedPromise =
-    ContentNodeResource.getCollection({ recommendations: '' }).fetch({}, true);
-  _updateChannelList(store);
-  recommendedPromise.only(checkSamePageId(store), (recommendations) => {
-    const pageState = { recommendations: recommendations.map(_contentState) };
-    store.dispatch('SET_PAGE_STATE', pageState);
-    store.dispatch('CORE_SET_PAGE_LOADING', false);
-    store.dispatch('CORE_SET_ERROR', null);
-  }, (error) => {
-    store.dispatch('CORE_SET_ERROR', JSON.stringify(error, null, '\t'));
-    store.dispatch('CORE_SET_PAGE_LOADING', false);
-  });
+  const id = 'current';
+  const sessionModel = SessionResource.getModel(id);
+  const sessionPromise = sessionModel.fetch();
+  sessionPromise.then((session) => {
+    const nextStepsPayload = { next_steps: session.user_id, channel: channelId };
```

Comment:
Oh, didn't see the bootstrapping.

We should access the bootstrapped data in exactly the same way, so that we can still use the same Promise machinery, regardless of how the page is loaded.